osi_environment.proto

```diff
@@ -324,4 +336,150 @@ message EnvironmentalConditions
         //
         optional uint32 seconds_since_midnight = 1;
     }
+
+    //
+    // \brief Description of a cloud layer.
+    //
+    message CloudLayer
+    {
+        // Properties of a cloud layer by discrete cloud layer levels.
+        //
+        optional FractionalCloudCover cloud_layer_level = 1;
+
+        // Defines the fractional cloud cover given by observation of total cloud amount in eights (oktas) of the sky.
+        //
+        // For visual alignment please see reference[1].
+        //
+        // The total degree of coverage indicates how much of the sky is covered with clouds altogether.
+        // 0 oktas means that there are no traces of clouds in the sky, \c
+        // 8 oktas means that the sky is completely covered with clouds and no sky blue can be recognized.
+        //
+        // \par References:
+        // https://cie.co.at/eilvterm/17-29-116
```

Comment:
Reference should be reworked, like described here:
https://opensimulationinterface.github.io/osi-documentation/#_guidelines_for_creating_literary_references